Source/Blazorise/Components/Table/Table.razor.cs

```diff
@@ -30,12 +30,22 @@ public partial class Table : BaseDraggableComponent
 
     private bool fixedHeader;
 
+    private bool fixedColumns;
+
     private string fixedHeaderTableHeight = "300px";
 
     private string fixedHeaderTableMaxHeight = "300px";
 
     private bool resizable;
 
+    private TableRow lastTableRow;
```

Comment:
hmm a state for the `lastTableRow`, could this affect anyone trying to do dynamic stuff with the table?